relay/polkadot/src/xcm_config.rs

```diff
@@ -202,6 +202,7 @@ pub struct XcmConfig;
 impl xcm_executor::Config for XcmConfig {
 	type RuntimeCall = RuntimeCall;
 	type XcmSender = XcmRouter;
+	type XcmRecorder = XcmPallet; // FAIL-CI @cisco please check
```

Comment:
@cisco i copied this everywhere minus the glutton. Please check that it is correct, otherwise please fix.

Reply:
This is correct, should be fine